### PR TITLE
menu added

### PR DESCRIPTION
**Pull request by javhaa07**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -41,6 +41,59 @@
           </div>
         </div>
       </div>
+
+      <nav class="header-menu navbar navbar-static-top">
```
Comment (khos-erdene) [inline index.html]: navbar-static-top gdeg class yund heregtei ve? shaardlagagui class ashiglasan bn

File: index.html [hunk 2]
```diff
@@ -41,6 +41,59 @@
...
+        <div class="container-fluid">
+          <div class="navbar-header">
```
Comment (khos-erdene) [inline index.html]: <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="#">Brand</a>
    </div>

iim bh ystoi bdiishd ygd iim aimar uurchilchiv

File: index.html [hunk 3]
```diff
@@ -41,6 +41,59 @@
...
+            <div class="mx-auto">
```
Comment (khos-erdene) [inline index.html]: ene yun shine class garaad irev bas ashiglaagui

File: index.html [hunk 4]
```diff
@@ -41,6 +41,59 @@
...
+              <a class="navbar-brand menu-logo" href="#"><img src="images/logo.png"></a>
```
Comment (khos-erdene) [inline index.html]: Logonii ar tald Sys&CoTech gsen ug ni bhgui biluu?

Comment (khos-erdene) [reply index.html]: Responsive deer logo ingej golloh ni zuw shiidel uu? Albaar ingesnuu? Her zohimjtoi haragdaj bn? logo ni hide hiigdeed Sys&CoTech gedgiin ar tald menug zadaldag button bval zugeer ym bishuu? Bas devsger ni tsagaan uchraas menunii dooguur zuraas tatah heregtei bsiin bishuu? Yg duiraj bgaa zagvar shigee

Comment (khos-erdene) [reply index.html]: Menug zadalhad hajuu taldaa genet garj irdeg bug bn. Bas zadarsan menunuud ni devsger ungutei esvel deeguur dooguuraa zuraastai ch ym uu bh ystoin bishuu?

File: index.html [hunk 5]
```diff
@@ -41,6 +41,59 @@
...
+          </div>
+          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar-collapse-menu" aria-expanded="false">
+            <span class="sr-only">Toggle navigation</span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+          </button>
+          <div class="collapse navbar-collapse" id="navbar-collapse-menu">
+            <ul class="header-menu-nav nav navbar-nav navbar-right">
+              <li><a href="index.html" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Нүүр</span></a></li>
```
Comment (khos-erdene) [inline index.html]: Dooshoo zadardaggui menu nuud deer yg dooshoo zadrah codenii ymnuud bicheed yvchiv!!!

File: index.html (hunk 4 above)
Comment (javhaa07) [reply index.html]: За ойлголоо.

File: index.html (hunk 5 above)
Comment (khos-erdene) [reply index.html]: Mouse ochihod uur ungutei darhad uur ungutei bolood bhiin iim bj bolkushd. Bas tsenher ymuu huh ygaan unguur yvii gj tohiroogui bil uu?

Comment (khos-erdene) [reply index.html]: Com deer haragdah dooshoo sunadag menunii deed tald genet bas iim uur ungu oroh biluu?

File: index.html [hunk 6]
```diff
@@ -41,6 +41,59 @@
...
+            </div>
+          </div>
+          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar-collapse-menu" aria-expanded="false">
+            <span class="sr-only">Toggle navigation</span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+          </button>
+          <div class="collapse navbar-collapse" id="navbar-collapse-menu">
+            <ul class="header-menu-nav nav navbar-nav navbar-right">
```
Comment (khos-erdene) [inline index.html]: classiin neree anhnaasaa goy nershilsen deer header-menu-nav gedgiin orond header-menu bhad hangalttai

File: index.html [hunk 7]
```diff
@@ -41,6 +41,59 @@
...
+                      </span>
+                    </div>
+                  </li>
+                </ul>
+              </li>
+              <li class="dropdown">
+                <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">MN </i><span class="caret"></span></span></a>
+                <ul class="dropdown-menu">
+                  <li>
+                    <li><a href="#">EN</a></li>
```
Comment (khos-erdene) [inline index.html]: ene deer MN hol nemegdeh ystoi shuu. Odoo hedii mn hel deer bgaa ch gsen zaaval haragddag. Olon heltei tom2 situudiig haraarai

File: index.html [hunk 8]
```diff
@@ -41,6 +41,59 @@
...
+                  <li><a href="#">Бусад</a></li>
+                </ul>
+              </li>
+              <li><a href="#">Мэдээ</a></li>
+              <li><a href="#">Холбоо барих</a></li>
+              <li class="dropdown">
+                <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false"><i class="fa fa-search"></i><span class="caret"></span></span></a>
+                <ul class="dropdown-menu">
+                  <li>
+                    <div class="input-group">
```
Comment (khos-erdene) [inline index.html]: inputtai hesgiig form tag-d hiigeed anhnaasaa yvaarai hailt hiihed uur huudsaas haih uchraas

File: css/custom.css [hunk 9]
```diff
@@ -31,9 +31,78 @@
   color: blue;
   background-color: #e7e7e7;
 }
+
+/* START nav-menu */
+.header-menu {
+  min-height: inherit !important;
+}
+.menu-logo {
+  height: 100px;
```
Comment (khos-erdene) [inline css/custom.css]: iimerhuu zurgan deer height css deer tavij ugduggui shuu. Html deer taviarai

File: css/custom.css [hunk 10]
```diff
@@ -31,9 +31,78 @@
...
+  margin-right: 20px;
+}
+.header-menu-nav>li>ul {
+  border-bottom-left-radius: 0;
+  border-bottom-right-radius: 0;
+  background-color: #f6f6f6;
+  border-left: 0;
+  border-right: 0;
+  border-bottom: 0;
+  border-top: 2px solid #ff4c72;
```
Comment (khos-erdene) [inline css/custom.css]: enenii daraalal ni top right bottom left gj bdag shuu teriig barimtlah heregtei. Eniig ingej olon bichej bsnaas
`border: 0;
 border-top: 2px solid #ungu;
`
ingej bichej bolno shuu

File: css/custom.css [hunk 11]
```diff
@@ -31,9 +31,78 @@
...
+.header-menu-nav>li>a {
+  padding: 40px 0;
+  text-transform: uppercase;
+  color: #222222;
+  font-family: 'Poppins', sans-serif;
+  font-weight: bold;
+  margin-right: 20px;
+}
+.header-menu-nav>li>ul {
+  border-bottom-left-radius: 0;
```
Comment (khos-erdene) [inline css/custom.css]: zuvhun ene 2 border-radiusiig avj hayj bgaanmuu. Buheldee border-radius-gui bolguul zugeer ym bishuu. border-radius: 0;

File: css/custom.css [hunk 12]
```diff
@@ -31,9 +31,78 @@
...
+  padding-bottom: 15px;
+  font-weight: bold;
+  color: #222222;
+}
+.header-menu-nav>li>ul>li>a:hover {
+  transition: 0.4s;
+  background-color: #29e663;
+  color: #fff;
+}
+.header-menu-nav>li>a:focus, .header-menu-nav>li>a:link {
```
Comment (khos-erdene) [inline css/custom.css]: :link :focus-iin css bicheh heregtei bsan uu?

File: css/custom.css [hunk 13]
```diff
@@ -31,9 +31,78 @@
...
+.header-menu-nav>li>ul>li {
+  margin: 0 !important;
+}
+.header-menu-nav>li>ul>li>a {
+  padding-top: 15px;
+  padding-bottom: 15px;
+  font-weight: bold;
+  color: #222222;
+}
+.header-menu-nav>li>ul>li>a:hover {
```
Comment (khos-erdene) [inline css/custom.css]: iim urt ym bichej bhaad .dropdown-menu a:hover geh maygaar handaad bichvel amar bishuu?

File: css/custom.css [hunk 14]
```diff
@@ -31,9 +31,78 @@
...
+  transition: 0.4s;
+  background-color: #29e663;
+  color: #fff;
+}
+.header-menu-nav>li>a:focus, .header-menu-nav>li>a:link {
+  background-color: transparent !important;
+  color: #29e663;
+}
+.header-menu-nav>li>a:hover {
```
Comment (khos-erdene) [inline css/custom.css]: buttsiin huvid ene nii hover ni ehleed bgd deer bichsen dropdownii hoveriig daraa ni bicheh ystoi bdag. Oilgomjgui bolgoj bn

File: css/custom.css [hunk 15]
```diff
@@ -31,9 +31,78 @@
...
+  color: #29a063;
+}
+.header-menu>.container-fluid>button {
```
Comment (khos-erdene) [inline css/custom.css]: .container-fluid -iin gj handah mash buruu shiidel bn. yr ni container container-fluidluu hamaagui ingej bichdeggui shuu. Ugaasaa htmlee buruu bichsen bnlee teriigee zaschuul .header-menu .navbar-header button gd handah ystoi

File: css/custom.css [hunk 16]
```diff
@@ -31,9 +31,78 @@
...
+  background-color: #29e663;
+  border-radius: 0;
+}
+.header-menu>.container-fluid>button>span {
+  background-color: #fff;
+}
+.header-menu>.container-fluid>.navbar-header>div {
+  margin: auto;
+  width: 90px;
```
Comment (khos-erdene) [inline css/custom.css]: hamaagui ingej urgun hucheer zaaj uguh demii bdag shuu. padding margin-aaraa shiideh heregtei esvel grid systemeer esve %-aar

File: css/custom.css [hunk 17]
```diff
@@ -31,9 +31,78 @@
...
+  color: #29e663;
+}
+.header-menu-nav>li>a:hover {
+  color: #29a063;
+}
+.header-menu>.container-fluid>button {
+  background-color: #29e663;
+  border-radius: 0;
+}
+.header-menu>.container-fluid>button>span {
```
Comment (khos-erdene) [inline css/custom.css]: ene bas container-fluid-ruu handaj bolku enenii doodoh ch bas

File: css/custom.css [hunk 18]
```diff
@@ -31,9 +31,78 @@
   color: blue;
   background-color: #e7e7e7;
 }
+
```
Comment (khos-erdene) [inline css/custom.css]: yr ni dandaa class > tag iim maygaar yvsan bn. > ene yund ashiglagddagiig mdeh uu? ygd .class tag{ } gj bichku bgaa bilee

File: css/custom.css (hunk 12 above)
Comment (tsmandax) [reply css/custom.css]: bootstrapiin hoveriig ni daraad

File: css/custom.css [hunk 19]
```diff
@@ -31,9 +207,135 @@
...
+    border: 0;
+  }
+  .navbar .nav .search-form-sm button {
+    outline-color: rgba(0, 55, 255, 0.5);
+    border-radius: 0;
+    border: 1;
+    background-color: #eeeeee;
+    color: #0055cc;
+  }
+  .navbar .nav .search-form {
```
Comment (khos-erdene) [inline css/custom.css]: ene hereggui uildel bn. html deeree hidden-xs gj bichsen ni deer

File: index.html [hunk 20]
```diff
@@ -16,33 +17,126 @@
...
+                <form action="search.html" method="GET" class="dropdown-menu">
+                  <div class="input-group">
+                    <input type="text" class="form-control" name="q" placeholder="Хайх...">
+                    <span class="input-group-btn">
+                      <button class="btn" type="submit"><i class="fa fa-search"></i></button>
+                    </span>
+                  </div>
+                </form>
+              </li>
+              <li class="hidden-lg hidden-md hidden-sm search-form-sm">
```
Comment (khos-erdene) [inline index.html]: search-form-sm gdeg classiin ner onovchgui bn. sm gdeg ni ipad tabletuudiig ilerhiildeg. Ene zuvhun mobile deer shalgaj bgaa uildel bzdee te?

**Pull request merged**